src/Features/Core/Portable/LanguageServices/AnonymousTypeDisplayService/AbstractAnonymousTypeDisplayService.cs

```diff
@@ -1,176 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
this type was not deleted.  but enough changed happened to it that github considers it a rewrite.